verisignmdns/src/main/java/denominator/verisignmdns/VerisignMDNSZoneApi.java

```diff
@@ -0,0 +1,31 @@
+package denominator.verisignmdns;
+
+import java.util.Iterator;
+
+import javax.inject.Inject;
+import javax.inject.Provider;
+
+import denominator.Credentials;
+import denominator.model.Zone;
+
+public final class VerisignMDNSZoneApi implements denominator.ZoneApi {
```

Comment:
The only public class in this package should be VerisignMDNSProvider. Please hide the others including this one.